### PR TITLE
[text analytics] prepare docstrings for release

### PR DESCRIPTION
**Pull request by iscai-msft**

### CODE REVIEW

File: sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 5.1.0b5 (Unreleased)
+## 5.1.0b5 (2020-02-10)
```
Comment (kristapratico) [inline sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md:3]: 😄 

```suggestion
## 5.1.0b5 (2021-02-10)
```

File: sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py [hunk 2]
```diff
@@ -1292,6 +1293,11 @@ class RecognizeEntitiesAction(DictMixin):
         `UnicodeCodePoint`, the Python encoding, is the default. To override the Python default,
         you can also pass in `Utf16CodePoint` or TextElements_v8`. For additional information
         see https://aka.ms/text-analytics-offsets
+    :ivar str model_version: The model version to use for the analysis.
```
Comment (kristapratico) [inline sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py:1296]: should we link here for model version?

https://docs.microsoft.com/azure/cognitive-services/text-analytics/concepts/model-versioning

File: sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md (hunk 1 above)
Comment (iscai-msft) [reply sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md:3]: lmao thanks

File: sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py (hunk 2 above)
Comment (iscai-msft) [reply sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py:1296]: Why not, thanks!

File: sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py [hunk 3]
```diff
@@ -1321,11 +1327,18 @@ class RecognizePiiEntitiesAction(DictMixin):
...
-    subset of the PII entity categories. Possible values include 'phi' or None.
+        subset of the PII entity categories. Possible values include 'phi' or None.
     :keyword str string_index_type: Specifies the method used to interpret string offsets.
         `UnicodeCodePoint`, the Python encoding, is the default. To override the Python default,
         you can also pass in `Utf16CodePoint` or TextElements_v8`. For additional information
         see https://aka.ms/text-analytics-offsets
+    :ivar str model_version: The model version to use for the analysis.
+    :ivar str domain_filter: An optional string to set the PII domain to include only a
+        subset of the PII entity categories. Possible values include 'phi' or None.
+    :ivar str string_index_type: Specifies the method used to interpret string offsets.
```
Comment (abhahn) [inline sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py:1338]: Just for future clarification, we are supposed to be duplicating the docs for keyword args and internal variables?

Comment (iscai-msft) [reply sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py:1338]: yeah, if the model is used as an input we include the keyword arg docs in addition to the internal variables

Comment (abhahn) [reply sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py:1338]: Ok, thanks for the info, I'll be sure to add both next time.  :)

**Review by kristapratico — APPROVED** [review]

**Review by abhahn — APPROVED** [review]

**Pull request merged**